viewer/packages/api/src/api-helpers/Image360ApiHelper.ts

```diff
@@ -125,20 +126,22 @@ export class Image360ApiHelper {
   }
 
   public async enter360Image(image360Entity: Image360Entity): Promise<void> {
-    const lastEntered360ImageEntity = this._interactionState.currentImage360Entered;
-    this._interactionState.currentImage360Entered = image360Entity;
-
-    if (lastEntered360ImageEntity === image360Entity) {
+    const lastClicked360ImageEntity = this._interactionState.currentImage360Clicked;
+    if (lastClicked360ImageEntity === image360Entity) {
       this._requestRedraw();
       return;
     }
+    this._interactionState.currentImage360Clicked = image360Entity;
```

Comment:
360 image enter can be called even without icon been clicked e.g enter into image when loading only 360 Images in Reveal, in this case, we should have `currentImage360Entered` instead of `currentImage360Clicked`?